PlayCover/View/PlayCoverApp.swift

```diff
@@ -50,14 +50,17 @@ struct PlayCoverApp: App {
                 .environmentObject(InstallVM.shared)
                 .environmentObject(AppsVM.shared)
                 .environmentObject(AppIntegrity())
-                .frame(minWidth: 720, minHeight: 650)
+                .frame(minWidth: 535, minHeight: 650)
```

Comment:
Change the width back please

Reply:
This is done intentionally to fit 3 columns of app icons perfectly.

Reply:
The 3 column setup does not look appealing.

Reply:
> The 3 column setup does not look appealing.

The user always have the option to upsize the window though.

Reply:
Please change it back.

Reply:
Still modified minimum width but it now fits 4 columns of apps perfectly without extra padding on the right.